app/templates/_service_attributes.html

```diff
@@ -1,28 +1,18 @@
-{% import "toolkit/summary-table.html" as summary %}
+{% from "govuk-frontend/components/summary-list/macro.njk" import govukSummaryList %}
 
 {% for section in service.summary_manifest %}
   {% if not section.is_empty %}
     {% if loop.index < 10 %}
-      <div class="scroll-tracking" id="0{{ loop.index }}-{{ section.slug }}">
+    <div class="scroll-tracking" id="0{{ loop.index }}-{{ section.slug }}">
     {% else %}
-      <div class="scroll-tracking" id="{{ loop.index }}-{{ section.slug }}">
+    <div class="scroll-tracking" id="{{ loop.index }}-{{ section.slug }}">
     {% endif %}
-      {{ summary.heading(section.name, id=section.slug) }}
-
-      {% call(question) summary.list_table(
-        section.questions,
-        caption=section.name,
-        field_headings_visible=False
-      ) %}
-        {% if not question.is_empty %}
-          {% call summary.row() %}
-            {{ summary.field_name(question.label) }}
-            {% with item_value = summary[question.type](question.filter_value|capitalize_first, question.assurance) %}
-              {{ (item_value | preserve_line_breaks) if question.type == 'textbox_large' else item_value }}
-            {% endwith %}
-          {% endcall %}
-        {% endif %}
-      {% endcall %}
+      <h2 id="{{ section.slug }}" class="govuk-heading-m govuk-!-font-size-27">{{ section.name }}</h2>
```

Comment:
Is the font size override here because that's what it used to look like, or because it looks better this way?

Reply:
The former initially, but I found with all the bold questions the extra size helps the heading stand out a bit.